locales/th/php-inline.json

```diff
@@ -1,124 +1,124 @@
-    "required_unless": "ช่องข้อมูลนี้เป็นต้องการนอกจาก :other อยู่ใน :values.",
-    "required_with": "ช่องข้อมูลนี้เป็นต้องการตอนที่ :values เป็นของขวัญ",
-    "required_with_all": "ช่องข้อมูลนี้เป็นต้องการตอนที่ :values เป็นของขวัญ",
-    "required_without": "ช่องข้อมูลนี้เป็นต้องการตอนที่ :values ไม่ใช่ของขวัญ",
-    "required_without_all": "ช่องข้อมูลนี้เป็นต้องการตอนที่ไม่ใช่เรื่องของ :values เป็นของขวัญ",
-    "same": "ค่าของช่องข้อมูลนี้จะต้องตรงกับค่าใดค่าหนึ่งจาก :other.",
+    "attached": "ข้อมูลถูกแนบในฟิลด์นี้แล้ว",
+    "before": "ต้องระบุวันที่ก่อน :date.",
+    "before_or_equal": "ฟิลด์นี้ต้องเป็นวันที่ :date หรือก่อนหน้านั้น",
+    "between.array": "ข้อมูลส่วนนี้ต้องมีค่ามากกว่า :min และไม่เกิน :max รายการ",
+    "between.file": "ไฟล์นี้ต้องมีขนาดมากกว่า :min และไม่เกิน :max กิโลไบต์",
+    "between.numeric": "ข้อมูลต้องมีค่ามากกว่า :min และไม่เกิน :max",
+    "between.string": "ฟิลด์นี้ต้องมีความยาวตัวอักษรมากกว่า :min และไม่เกิน :max ตัวอักษร",
+    "boolean": "รองรับการระบุข้อมูล จริง(true) หรือ เท็จ(false) เท่านั้น",
+    "can": "ฟิลด์นี้มีค่าที่ไม่ได้รับอนุญาต",
+    "confirmed": "การยืนยันข้อมูลไม่ตรงกัน",
+    "country": "ข้อมูลที่ระบุไม่ใช่ประเทศที่ถูกต้อง",
+    "date": "ฟิลด์นี้ต้องระบุเป็นวันที่เท่านั้น",
+    "date_equals": "ข้อมูลที่ระบุต้องมีวันที่เท่ากับ :date",
+    "date_format": "ระบุข่อมูลไม่ตรงกับรูปแบบ :format ที่รองรับ",
+    "decimal": "ข้อมูลที่ระบุต้องเป็นเลขทศนิยม :decimal ตำแหน่ง",
+    "declined": "ต้องปฏิเสธฟิลด์นี้",
+    "declined_if": "ต้องปฏิเสธฟิลด์นี้หาก :other มีค่าเท่ากับ :value",
+    "different": "ข้อมูลของฟิลด์นี้และ :other ต้องแตกต่างกัน",
+    "digits": "ฟิลด์นี้ต้องมีเลขจำนวน :digits หลัก",
+    "digits_between": "ฟิลด์นี้ต้องมีความยาวมากกว่า :min และไม่เกิน :max",
+    "dimensions": "ขนาดรูปภาพไม่ถูกต้อง",
+    "distinct": "ข้อมูลที่ระบุมีค่าที่ซ้ำกัน",
+    "doesnt_end_with": "ฟิลด์นี้ไม่ควรลงท้ายด้วย: :values",
+    "doesnt_start_with": "ฟิลด์นี้ไม่ควรเริ่มต้นด้วย: :values",
+    "email": "โปรดระบุอีเมลที่ถูกต้อง",
+    "ends_with": "ข้อมูลที่ระบุควรลงท้ายด้วย: :values",
+    "enum": "ฟิลด์ที่เลือกไม่ถูกต้อง",
+    "exists": "ฟิลด์ที่เลือกไม่ถูกต้อง",
+    "file": "เนื้อหาต้องเป็นรูปแบบไฟล์เท่านั้น",
+    "filled": "จำเป็นต้องระบุข้อมูลในฟิลด์นี้",
+    "gt.array": "ฟิลด์นี้ต้องมีข้อมูลมากกว่า :value รายการ",
+    "gt.file": "ไฟล์ต้องมีขนาดมากกว่า :value กิโลไบต์",
+    "gt.numeric": "ฟิลด์นี้ต้องมีค่ามากกว่า :value",
+    "gt.string": "ฟิลด์นี้ต้องมีความยาวมากกว่า :value ตัวอักษร",
+    "gte.array": "ฟิลด์นี้ต้องมีข้อมูล :value รายการขึ้นไป",
+    "gte.file": "ไฟล์ต้องมีขนาด :value กิโลไบต์ขึ้นไป",
+    "gte.numeric": "ฟิลด์นี้ต้องมีค่า :value ขึ้นไป",
+    "gte.string": "ฟิลด์นี้ต้องมีความยาว :value ตัวอักษรขึ้นไป",
+    "image": "ข้อมูลส่วนนี้ต้องเป็นรูปภาพ",
+    "in": "ฟิลด์ที่เลือกไม่ถูกต้อง",
+    "in_array": "ไม่มีค่าหรือข้อมูลนี้อยู่ใน :other",
+    "integer": "ต้องระบุฟิลด์นี้เป็นเลขจำนวนเต็มเท่าน้้น",
+    "ip": "ต้องระบุข้อมูลเป็น IP Address ที่ถูกต้องเท่านั้น",
```

Comment:
I think we should translate `IP Address` to Thai, so it should be `ต้องระบุข้อมูลเป็นไอพีแอดเดรสที่ถูกต้องเท่านั้น`